node/api/channels_api_usage.md

```diff
@@ -1136,6 +1140,179 @@ curl 'http://127.0.0.1:3013/v2/transactions/th_2qkN973cNJiejXVJoXkXbttf1iKetWJCS
 ```
 if the `block_hash` is `none` - then the transaction is still in the mempool.
 
+## Channel solo close
+It is possible to close the channel unilaterally, e.g. if the other party has
+disconnected and is expected never to return. The channel fsm can be asked
+to generate a `channel_close_solo` transaction and post it on-chain. The
+resulting transaction will include the latest mutually signed offchain state,
+or the empty string, indicating that the latest state is what's on the chain.
+
+The `channel_close_solo` transaction only needs a single signature, and is described in more detail in [this section](#channel-solo-close).
```

Comment:
Can a wrapping generic account meta tx be provided instead of a signature?

It would be nice to consider signature-vs-generic-accounts in new docs. I expect other docs needs review fine-tuning.

Reply:
@lucafavatella I consider this as part of my PR [SC FSM: GA awareness](https://www.pivotaltracker.com/story/show/165413555)